samples/net/updatehub/README.rst

```diff
@@ -81,6 +83,7 @@ Using this server the user need create own ``overaly-prj.conf`` setting the opti
 :option:`CONFIG_UPDATEHUB_CE` with true. If the user will use polling mode on
 UpdateHub need too set the option :option:`CONFIG_UPDATEHUB_POLL_INTERVAL` with the period of
 your preference, remembering that the limit is between 0 minute until 43200 minutes(30 days).
+And this server does not use DTLS, so you do not need to add the ``overlay-dtls.config``.
 
```

Comment:
BTW, could there be simple instructions how to create the .der files if user wants use own files instead of the supplied ones.

Reply:
The docs should be really clear that the supplied keys should only ever be used for testing. Aside from this, since the private key is compiled into the image, it means that every device will require a separate image. Really, the key needs to be stored in some kind of protected storage on the device. This is one problem that TF-M seeks to address.

Reply:
@d3zd3z 
> The docs should be really clear that the supplied keys should only ever be used for testing.

Agreed.

> Aside from this, since the private key is compiled into the image, it means that every device will require a separate image. Really, the key needs to be stored in some kind of protected storage on the device. This is one problem that TF-M seeks to address.

Could you shed some light on the TF-M? Not sure I follow you here.

Reply:
Zephyr doesn't currently have anywhere to store a secret, safely. This kind of API should come in some day, though, and TF-M is one solution that may provide that.

Reply:
Ok, in fact, for production I'd expect it to be written during the manufacture so it can be individual and not part of the firmware.

What do you think we should do now? Leave it as is?